src/Parser/Shaman/Elemental/Modules/Talents/LightningRod.js

```diff
@@ -21,21 +20,33 @@ class LightningRod extends Analyzer {
     combatants: Combatants,
   };
 
+  lightningRodDamage=0;
+
   on_initialized() {
     this.active = this.combatants.selected.hasTalent(SPELLS.LIGHTNING_ROD_TALENT.id);
   }
 
+  on_byPlayer_damage(event) {
+    if (event.ability.guid===SPELLS.LIGHTNING_ROD_DAMAGE.id){
+      this.lightningRodDamage+=event.amount;
+    }
+  }
+
   get rawUpdate() {
```

Comment:
I know this isn't part of your changes, but it's bothering me that this neither 'raw' nor an 'update'. Probably should read `percentUptime`